README.md

```diff
@@ -211,6 +212,19 @@ scrape({
 ```
 Scrape function resolves with array of [Resource](https://github.com/s0ph1e/node-website-scraper/blob/master/lib/resource.js) objects which contain `metadata` property from `httpResponseHandler`. 
 
+#### resourceStorage
+Class which saves [Resources](https://github.com/s0ph1e/node-website-scraper/blob/master/lib/resource.js), should have methods `saveResource` and `removeSavedResources` which return Promises. Use it to save files where you need: to dropbox, amazon S3, existing directory, etc. By default all files are saved in local file system to new directory passed in `directory` option (see [lib/resource-storage.js](https://github.com/s0ph1e/node-website-scraper/blob/master/lib/resource-storage.js)).
+```javascript
+scrape({
+  urls: ['http://example.com/'],
+  directory: '/path/to/save',
+  resourceStorage: class MyResourceStorage {
+  	saveResource (resource) {/* code to save file where you need */}
+  	removeSavedResources() {/* code to remove all previously saved files in case of error */}
```

Comment:
It's not clear what `remove all previously saved files in case of error` means.
What the error? Why we have to remove files?

`removeSavedResources` no any mentions about error. Is it used in any other cases?